sdk/keyvault/Azure.Security.KeyVault.Administration/CHANGELOG.md

```diff
@@ -1,14 +1,18 @@
 # Release History
 
-## 4.0.0-beta.4 (Unreleased)
+## 4.0.0-beta.4 (2021-02-10)
 
 ### Added
 
 - Support in `KeyVaultAccessControlClient` to create, update, and delete custom role definitions.
 
+### Changed
```

Comment:
I believe we are supposed to use "New Features" - https://azure.github.io/azure-sdk/policies_releasenotes.html

Reply:
Meant to put this comment for Added, but Changed also doesn't seem like one of the allowed headers.

Reply:
Thanks. That refers to release notes, however. This follows the current changelog guidance: https://azure.github.io/azure-sdk/policies_releases.html#2-change-logs--migration-guides

/cc @jsquire

Reply:
Ah ok looks like you are right.

Reply:
I'm not sure, to be honest.  Adrian had been the one that asked me to keep the changelog form in accordance to the release notes guidance.     

@AlexGhiondea,  @annelo-msft:   Do either of your folks have an authoritative answer on whether we're applying two different formats between change logs and release notes and, if not, which of these policies takes precedence?